### DISCUSSION TITLE
Error message repeatedly occuring when attempting to open site

### DISCUSSION THREAD

**Discussion by cececrocker** [CollectionBuilder/collectionbuilder.github.io#39] [Q&A]
I've followed all the tutorials and read all the documentation I can get my hands on -- the site that I'm attempting to make, a collection of family recipes, just doesn't seem to be working. Whenever I click on the generated url, I just get a 404 error. This is the link in question: cececrocker.github.io/collection-of-crocker-family-recipes/ 
I'm not sure if it's something in the config.yml or something else entirely. Any suggestions?

**Comment by dcnb** [comment]
hi @cececrocker

It seems like you've turned on github pages, so  I'm not exactly sure
what's going on. It's good practice not to use capital letters in your
repository name. One suggestion would be to rename the repository so that
all the capital letters are now lowercase.

I took a look at your collection as well. There are several issues:

1) Your metadata file "Family Recipes.csv" has a space in it. You should
add a dash rather than a space, which will make it connect with the value
you put in the _config.yml file.
   - it's also good practice here to only use lowercase, so you might just
rename the file to familyrecipes.csv and put "familyrecipes" as your
`metadata` value in the config file, but it will work if you just add a
dash.
2) Your filenames do not match the files you have in the objects folder. It
looks like you renamed the files after their objectids, so in your
spreadsheet, you can just replace the filename option with your objectid
and then add .jpg to each one. You could use a formula to do this,
something like (if objectid is column A) -->   =A2&".jpg"  and then you can
drag that down the row.

It also looks like you have an extra line at the top of your csv listing
the title of the filename, you'll want to delete that.

I'll see what @evanwill has to say about the website not working, but I bet
if you make the fixes above and change the case of the repository, that
will likely clear things up. You might even do rename it something simpler,
like "crocker-recipes."  You can rename the repository at the top of the
setttings page in the repository.

Cheers!
Devin

**Reply by cececrocker** [reply]
Thank you both for the help! The website itself is now working, but the objects themselves aren't showing up now? And when I try to download the csv table, it just comes up with n/a and the link to the github. Is there any fix to this??

**Reply by evanwill** [reply]
@cececrocker the issue is extra lines on top of your [metadata file](https://github.com/cececrocker/collection-of-crocker-family-recipes/blob/main/_data/familyrecipes.csv)

- visit the [metadata file](https://github.com/cececrocker/collection-of-crocker-family-recipes/blob/main/_data/familyrecipes.csv)
- click the edit pencil
- delete the first two lines:

```
Family Recipes
Copy of Crocker Family Recipes Metadata chart - Sheet1,,,,,,,,,,,,,,,,,,
```

So that in the end the first line of your metadata file should be the column headers (which is currently the 3rd line):

```
objectid,filename,youtubeid,vimeoid,title,creator,date,description,subject,location,latitude,longitude,source,identifier,type,format,language,rights,rightsstatement
```

**Reply by cececrocker** [reply]
I did this, and I don't think that it worked. There's still n/a and when I try to download the data set, it just shows a section of the heading

**Reply by evanwill** [reply]
@cececrocker the first line of your CSV is still `familyrecipes`, so that needs to be deleted!

**Reply by cececrocker** [reply]
Thank you! Now the only thing that doesn't seem to be updating is the About page. I've updated the Readme file but it doesn't seem to translate over into the About page.

**Reply by owikle** [reply]
@cececrocker Great to hear it's working now! The Readme file doesn't update the About page. You'll need to edit the about.md file in the repository's "pages" folder in order to update your About content. Here's a link to the instructions in the docs: https://collectionbuilder.github.io/cb-docs/docs/pages/interpretive/

**Reply by cececrocker** [reply]
Hello! Thank you for the help, but now the About page isn't carrying over the information that I've put in. I deleted the bottom comment and applied the changes that were needed.

**Reply by owikle** [reply]
@cececrocker It looks like the About page is displaying your content now? https://cececrocker.github.io/collection-of-crocker-family-recipes/about.html. Sometimes it can take a minute or two for GitHub Pages to update the site with your changes, so that might have been what was going on. Let us know if you run into other questions!

**Comment by evanwill** [comment]
@cececrocker I like the recipe collection idea!  

I was just looking at your question when @dcnb replied as well--> so basically just said the same thing about metadata here: https://github.com/cececrocker/Collection-of-Crocker-Family-Recipes/issues/1

As for why it isn't building--I really don't see any thing. I am not sure. I do not believe the letter case should break it. I think try fixing the metadata issues first, then let's see if it builds after that? Let us know when you have tried that. 

Good luck!

**Reply by evanwill** [reply]
p.s. It does build correctly on my local machine with the metadata fixes, so you are close! Just not sure what the github pages issue is yet.

**Comment by evanwill** [comment]
@cececrocker on the github pages build issue (once the metadata is fixed), my first guess so far would be to make sure that you have the correct branch and folder selected in the Pages settings. Check that in Settings > Pages, under "Source" you have "Branch: main" and "/ (root)" selected and saved. If you have a folder other than "(root)" it will not build. See example below:

![ghpages](https://user-images.githubusercontent.com/13864643/144896928-f287393f-d7a4-4da4-b3a7-2e7239b93ff9.PNG)
